lib/erlef_web/plugs/json_events.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ErlefWeb.Plug.JsonEvents do
+  import Plug.Conn
+
+  def init(default), do: default
+
+  def call(conn, _default), do: assign(conn, :events, all_events())
+
+  defp all_events do
+    events =
+      Enum.map(Erlef.Events.Repo.all(), fn e ->
```

Comment:
The transforms on some of this data might be better in the javascript : pros and cons. Works for now.